src/main/java/com/sounganization/botanify/domain/chat/components/ChatDataCleanupScheduler.java

```diff
@@ -0,0 +1,54 @@
+package com.sounganization.botanify.domain.chat.components;
+
+import com.sounganization.botanify.domain.chat.repository.ChatMessageRepository;
+import com.sounganization.botanify.domain.chat.repository.ChatRoomRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class ChatDataCleanupScheduler {
+    private final ChatMessageRepository chatMessageRepository;
+    private final ChatRoomRepository chatRoomRepository;
+
+    @Scheduled(cron = ChatMessageConstants.CLEANUP_CRON)
```

Comment:
Thinking of benefits to collect all @Scheduled jobs in a single place (like common/service/SchedulerService.java)

Not that crucial, nevermind
maybe I could do some refactoring later

Reply:
That would be great. I'll make the codebase more organized~^^